src/go/cmd/update-collection-v3/main.go

```diff
@@ -57,6 +58,7 @@ var migrationDirectoriesAndFunctions = map[string]migrateFunc{
 	"events":                events.Migrate,
 	"disable-thanos":        disablethanos.Migrate,
 	"tracing-replaces":      tracingreplaces.Migrate,
+	"logs":                  logs.Migrate,
```

Comment:
The migration name - "logs" - is a bit too general to my taste. How about something more like "simplify-logs-metadata-config"?

Reply:
The intent is for this migration to contain all log-related changes. Do you think it's better to split that across multiple migrations?

Reply:
I thought we were going to have a separate migration for each change...

Reply:
I think this is [one change](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2626) for now, but in case other changes we will need other migrations.

Reply:
I'm fine changing the name to something more specific.